### PR TITLE
Fix #4591: Use text field instead of tags field for $:/config/NewJournal/Tags ...

### LINKED ISSUE #4591: [BUG] Journal tag pill dropdown list includes the system tiddler "$:/config/NewJournal/Tags"
**Bug description**
Clicking the Journal tag pill produces a dropdown list that wrongly includes a single system tiddler as the first tiddler titled: "$:/config/NewJournal/Tags" in TW5 v5.1.22 This behaviour seems to be restricted to only that action. Using list generating macros correctly excludes the system tiddler. Changing that system tiddler to remove the behavior breaks the creation of new Journal tiddlers.

**To Reproduce**
1. Create a non-system tiddler tagged Journal
2. Click on Journal tag pill
3. First entry shows ""$:/config/NewJournal/Tags""

**Expected behavior**
"$:/config/NewJournal/Tags" should not appear in the dropdown list.

**Screenshots**
EWNXSr0X0AAxAtX

### PR DESCRIPTION
**Pull request by BurningTreeC**
... and $:/config/NewTiddler/Tags

The tags input in $:/ControlPanel/ Info/ Basics/ is replaced by simple text input fields

### CODE REVIEW

File: core/language/en-GB/NewJournalTags.tid [hunk 1]
```diff
@@ -1,2 +1,2 @@
 title: $:/config/NewJournal/Tags
-tags: Journal
```
Comment (pmario) [inline core/language/en-GB/NewJournalTags.tid:2]: hmmm, pls use 

```
title: $:/config/NewJournal/Tags

Journal
```

Comment (BurningTreeC) [reply core/language/en-GB/NewJournalTags.tid:2]: done @pmario

**Comment by BurningTreeC** [comment]
bump. I think this should be changed

**Comment by Jermolene** [comment]
Thanks @BurningTreeC

**Pull request merged**